django_ibmi/creation.py

```diff
@@ -262,31 +246,30 @@ def create_test_db( self, verbosity = 0, autoclobber = False , keepdb=False,seri
     # Method to destroy database. For Jython nothing is getting done over here.
     def destroy_test_db( self, old_database_name, verbosity = 0 ):
         print ("Destroying Database...")
-        if not _IS_JYTHON:
-            kwargs = self.__create_test_kwargs()
-            if( old_database_name != kwargs.get( 'database' ) ):
-                kwargsKeys = kwargs.keys()
-                if ( kwargsKeys.__contains__( 'port' ) and 
-                    kwargsKeys.__contains__( 'host' ) ):
-                    kwargs['dsn'] = "DATABASE=%s;HOSTNAME=%s;PORT=%s;PROTOCOL=TCPIP;" % ( 
-                             kwargs.get( 'database' ),
-                             kwargs.get( 'host' ),
-                             kwargs.get( 'port' )
-                    )
-                else:
-                    kwargs['dsn'] = ''
-                if kwargsKeys.__contains__( 'port' ):
-                    del kwargs['port']
-                if verbosity > 1:
-                    print ("Droping Test Database %s" % ( kwargs.get( 'database' ) ))
-                Database.dropdb( **kwargs )
-                
-            if( djangoVersion[0:2] <= ( 1, 1 ) ):
-                settings.DATABASE_NAME = old_database_name
-                settings.PCONNECT = True
+        kwargs = self.__create_test_kwargs()
+        if( old_database_name != kwargs.get( 'database' ) ):
+            kwargsKeys = kwargs.keys()
+            if ( kwargsKeys.__contains__( 'port' ) and
+                kwargsKeys.__contains__( 'host' ) ):
+                kwargs['dsn'] = "DATABASE=%s;HOSTNAME=%s;PORT=%s;PROTOCOL=TCPIP;" % (
+                         kwargs.get( 'database' ),
+                         kwargs.get( 'host' ),
+                         kwargs.get( 'port' )
+                )
             else:
-                self.connection.settings_dict['NAME'] = old_database_name
-                self.connection.settings_dict['PCONNECT'] = True
+                kwargs['dsn'] = ''
+            if kwargsKeys.__contains__( 'port' ):
+                del kwargs['port']
+            if verbosity > 1:
+                print ("Droping Test Database %s" % ( kwargs.get( 'database' ) ))
+            pyodbc.dropdb( **kwargs )
```

Comment:
pyodbc doesn't have a dropdb function

Reply:
Was focusing on just changing the ibm_db stuff to python and dealing with those in a later PR, should I deal with those in the same PR?